share/revng/pipelines/revng-pipelines.yml

```diff
@@ -506,15 +506,17 @@ Branches:
       - Name: embed-statement-comments
         Pipes:
           - Type: embed-statement-comments
-            UsedContainers: [module.bc.zstd]
+            UsedContainers: [functions.bc.zstd]
       - Name: decompile
         Pipes:
           - Type: helpers-to-header
-            UsedContainers: [module.bc.zstd, helpers.h]
+            UsedContainers: [functions.bc.zstd, helpers.h]
           - Type: model-to-header
+            # **ivan** shouldn't we move this further up the pipeline?
```

Comment:
Use `WIP` for comments that should not be merged.
Don't let your name get merged.